src/DynamoCore/Logging/IAnalyticsClient.cs

```diff
@@ -15,6 +15,8 @@ public enum Categories
         Command,
         FileOperation,
         SearchUX,
+        Preferences,
```

Comment:
I think so, `"Just as with any constant, all references to the individual values of an enum are converted to numeric literals at compile time. "` (https://msdn.microsoft.com/en-us/library/sbbt4032.aspx)